src/util/flssUtils.js

```diff
@@ -64,6 +65,26 @@ export async function fetchFacets(
 	}
 }
 
+/**
+ * Fetches list of categories from FLSS
+ *
+ * @param {Object} apollo The Apollo client instant
+ * @returns {Promise<Array<Object>>} Promise for category name/id
+ */
+export async function fetchCategories(
```

Comment:
This file is unit tested, so we'll need tests for this new method, following a similar pattern to the other methods in this file.